### PR TITLE
Add ipython kernel option to User Guide

### PR DESCRIPTION
**Pull request by Astro-Kirsty**
Resolves https://github.com/gammapy/gammapy/issues/5231

I also adapted a new way for the bash code in line with the new sphinx, this shows the 'release' correctly

### CODE REVIEW

**Review by bkhelifi — DISMISSED** [review]
Thanks @Astro-Kirsty. It was indeed missing!

File: docs/getting-started/quickstart.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can download the Gammapy tutorial notebooks and the example
 datasets. The total size to download is ~180 MB. Select the location where you
 want to install the datasets and proceed with the following commands:
 
-.. code:: bash
+.. substitution-code-block:: console
```
Comment (registerrier) [inline docs/getting-started/quickstart.rst:34]: why is this change required? This is still giving the correct expected results no?

File: docs/getting-started/quickstart.rst [hunk 2]
```diff
@@ -57,12 +57,24 @@ variable directly in your shell with:
...
+Jupyter
+-------
+Once you have activated your gammapy environment you can start
+a notebook server by executing::
 
     cd notebooks
     jupyter notebook
 
+
+Another option is to utilise the ipykernel functionality of Jupyter Notebook:
```
Comment (registerrier) [inline docs/getting-started/quickstart.rst:70]: Maybe this should specify that you need an environment with jupyter installed besides the gammapy one.

**Review by registerrier — COMMENTED** [review]
Thanks @Astro-Kirsty ! Looks good. See inline comments.

File: docs/getting-started/quickstart.rst (hunk 1 above)
Comment (Astro-Kirsty) [reply docs/getting-started/quickstart.rst:34]: If you look at the [dev version](https://docs.gammapy.org/dev/getting-started/index.html#recommended-setup), the release is showing as `|release|`

Comment (registerrier) [reply docs/getting-started/quickstart.rst:34]: Indeed

File: docs/getting-started/quickstart.rst (hunk 2 above)
Comment (Astro-Kirsty) [reply docs/getting-started/quickstart.rst:70]: Not sure what you mean, as this is just describing how to set up the kernel, which needs to be done in the gammapy environment?

Comment (bkhelifi) [reply docs/getting-started/quickstart.rst:70]: @Astro-Kirsty, something like that is better? 
```
An other option is to utilise the ipykernel functionality of Jupyter Notebook. It offers the possibility to choose a kernel from a pre-defined list.
In order to create kernels in such list, here are the command lines:
```
You can make some adaptation by making an inline proposal ;) thanks...

**Review by QRemy — APPROVED** [review]
Thanks @Astro-Kirsty !
